CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [v0.3.19] - ??
```

Comment:
Is this supposed to be a date?

Reply:
I wasn't sure if we could get the release done today, so I didn't put in a date yet. I can make it 2019-31-12, but then we should make sure we get it through today. ;)

Reply:
Let's put it this way, I'm not going to r+ without a date ;)

bors r+

Reply:
You [did before](https://github.com/japaric/xargo/pull/270), but fair. ;)

Reply:
I guess I was hoping for some CI that would fix it in post